ftw/simplelayout/staging/staging.py

```diff
@@ -303,7 +303,7 @@ def _apply_children(self, source, target, condition=None, uuid_map=None):
                 target_child = target_children_map.pop(target_uid)
                 self._apply_children(source_child, target_child, uuid_map=uuid_map)
             else:
-                target_child = self._copy_new_obj(source_child, target)
```

Comment:
This is the only place where we used `_copy_new_obj` so you can remove that method.

Reply:
👍  I'll go after the tests :-( seems like I broke quite a bit